tests/Common/TracerTestTrait.php

```diff
@@ -149,6 +149,37 @@ public function inWebServer($fn, $rootPath, $envs = [], $inis = [])
         return $this->parseTracesFromDumpedData();
     }
 
+    /**
```

Comment:
I added this to the trait so we can mixin different test scenarios (web, cli) in the same test. The alternative would be to have 2 tests (one extending from `WebFrameworkTest` and the other from `CLITest`) but this would be less readable in my opinion.

Also, I think we should consider removing `CLITestCase` in favor of `IntegrationTestCase` + this function, but this is a longer conversation beyond the scope of this PR.